tests/e2e/utils/set-auth-token.js

```diff
@@ -0,0 +1,14 @@
+
+import { wpApiFetch } from './wp-api-fetch';
```

Comment:
Empty line before and no `Internal dependencies` docblock (mandatory anyway once #217 lands).